src/envConfig.ts

```diff
@@ -235,6 +235,14 @@ export const asEnvConfig = asObject({
       appId: asOptional(asString, 'edge')
     }).withRest
   ),
+  RANGO_INIT: asCorePluginInit(
+    asObject({
+      appId: asOptional(asString, 'edge'),
+      rangoApiKey: asOptional(asString, ''),
```

Comment:
please send apikey over Discord

Reply:
AFAIK , it's already sent to you by Martin.